src/app/resources/resources.component.ts

```diff
@@ -76,4 +117,23 @@ export class ResourcesComponent implements OnInit, AfterViewInit {
     };
   }
 
+  deleteSelected() {
+    const DeleteAll = confirm('Are you sure you want to delete?');
```

Comment:
@razu9861  I think we can reuse the existing dialog delete componet. So, could you please check it once ? And also for deleting one item , the message is display "Are you sure you want to delete?" I think for deleting one item it should be the same as we have done in other component (Eg; nation) . For deleting All or more than one item , it should display "Are you sure you want to delete all ? " or should display the deleting item name .Please check it once.